javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,49 @@
+//Part one homework
+console.log("My username on freeCodeCamp is: Nelia11.")
+
+//
+let yearOfBirth = 1996;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}.`)
+
+//Age-ify (A future age calculator) 
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027
+let dogYear = 70;
+let shouldShowResultInDogYears = true; // can be changed to false
+if (shouldShowResultInDogYears) {
+    console.log(`Your dog will be ${dogYear} dog years old in ${dogYearFuture}.`)
```

Comment:
In this case I think you should calculate your dogs age, because in this program it will always be 70 no matter what dates you would specify